File: generator/sources/java/com/kaltura/client/KalturaApiException.java
package com.kaltura.client;

public class KalturaApiException extends Exception {

	private static final long serialVersionUID = 6710104690443289367L;

	public KalturaApiException() {
		super();
	}
	
	public KalturaApiException(String message) {
		super(message);
	}
}

File: generator/sources/csharp/KalturaClient/KalturaAPIException.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Kaltura
{
    public class KalturaAPIException : ApplicationException
    {
        #region Private Fields
        private string _Code;
        #endregion

        #region Properties
        public string Code
        {
            get { return this._Code; }
        }
        #endregion

        public KalturaAPIException(string code, string message): base(message)
        {
            this._Code = code;
        }
    }
}
